src/core/README.md

```diff
@@ -29,7 +29,7 @@ rules tailored to our needs (e.g. `byteSize`, `duration` etc.). That means that
 by the "legacy" Kibana may be rejected by the `core` now.
 
 ### Logging
-`core` has its own [logging system](./server/logging/README.md) and will output log records directly (e.g. to file or terminal) when configured. When no 
+`core` has its own [logging system](./server/logging/README.mdx) and will output log records directly (e.g. to file or terminal) when configured. When no 
```

Comment:
Q: How does this link work on a normal MD viewer (i.e.: GitHub's viewer)?

I mean, it will _work_ by taking us to `./server/logging/README.mdx`, but [it doesn't render OK](https://github.com/TinaHeiligers/kibana/blob/logging-docs/src/core/server/logging/README.mdx) on Github.

Reply:
> I mean, it will work by taking us to ./server/logging/README.mdx, but it doesn't render OK on Github.

Missed this comment, but @afharo put it much more succinctly than I did 😄

Reply:
Fair points, My bad for not checking that before changing the links. I'll change them back.

Reply:
I've [changed the syntax](https://github.com/TinaHeiligers/kibana/blob/logging-docs/src/core/server/logging/README.mdx) for links and the `Note` section on appender additivity back to the original format.